src/components/checkbox/checkbox.tsx

```diff
@@ -65,34 +57,37 @@ export class CheckBox extends React.Component<CheckBoxProps, CheckBoxState> {
                 onClick={this.handleChange}
                 style-state={styleState}
                 role="checkbox"
-                aria-checked={indeterminate ? 'mixed' : value}
+                aria-checked={this.props.indeterminate ? 'mixed' : this.props.value}
             >
                 <input
```

Comment:
Seems to be missing the required prop? There should also be a test for this

Reply:
This was actually on purpose... we need to have some product discussions regarding `required` behavior and validation in general